kernel/build/build_deb

```diff
@@ -88,11 +88,17 @@ echo "*****"
 # Clean the git repo
 git clean -fxd
 
+# Install build dependencies
+sudo apt-get install -y bc
+
 # Prep the config
 echo "Preparing the kernel config"
-cp "$(dirname $0)/kernel-config-deb.basic" .config
+wget https://raw.githubusercontent.com/ceph/ceph-build/master/kernel/build/kernel-config-deb.basic -O .config
```

Comment:
Looks like the jjb for that plugin is here: http://docs.openstack.org/infra/jenkins-job-builder/builders.html#builders.config-file-provider

That approach would require manually entering that config into the jenkins UI though, unless I'm missing something. I'd much rather have that in source control then manually imported into the jenkins UI.